go/cmd/socket.go

```diff
@@ -159,6 +135,6 @@ func init() {
 	socketCreateCmd.Flags().StringVarP(&password, "password", "", "", "Password, required when protected set to true")
 	socketCreateCmd.Flags().StringVarP(&socketType, "type", "t", "http", "Socket type, defaults to http")
 	socketCreateCmd.MarkFlagRequired("name")
-	socketDeleteCmd.Flags().StringVarP(&socketID, "id", "i", "", "Socket ID")
-	socketDeleteCmd.MarkFlagRequired("id")
+	socketDeleteCmd.Flags().StringVarP(&socketID, "socket_id", "i", "", "Socket ID")
```

Comment:
I think it's better to use the -s as the short flag, now it's -i.

Reply:
Agree, I will change it. It's then also consistent with the tunnel cmd which uses -s for socket_id